activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/cmd/SaveTaskCmd.java

```diff
@@ -61,15 +61,7 @@ public Task execute(CommandContext commandContext) {
       }
 
     } else {
-
```

Comment:
Probably the logic should remain in SaveTaskCmd similar to SetTaskPriorityCmd and SetTaskDueDateCmd.
What happens if the listener changes the due date, priority, or category of a task?

Reply:
Two good questions, hope that I can provide two great answers
**SaveTaskCmd**
`SaveTaskCmd` has the same logic, only changed the way we fetch the historical task (now on the `TaskEntityManager`). If the comment was to make `SaveTaskCmd` more like `SetTaskPriorityCmd`, think we need to compare with historical task since there are a lot of fields to check/update. The lesser fields we need to update, the better.
Also, seems like each `*.record*` method checks inside for the `AUDIT` level, therefore we can ditch that check on this class.

**Other operations**
I've only fixed what was described in the ticket since there are a lot fields to cover. Which fields should I consider? (I'm going to change the ones on your comment)

Reply:
A runtime task could be retrieved and modified (multiple modifications such as name, due date, priority, etc) and then saved using `taskService.save` which will call the `SaveTaskCmd` to save. Any changes should be possible to captured.

Reply:
I've got a few questions from my solution which I would like to discuss with you @balsarori , namely the `*UpdaterHelper` classes